README.md

```diff
@@ -0,0 +1,50 @@
+# **The Debuggers** 👩‍💻👨‍💻🕷
+
+## **Welcome to Our Project!** ![Alt Text](assets/hi.gif)
+
+Welcome to **The Debuggers'** official repository! We’re a group of passionate
+students collaborating on exciting programming projects, solving complex problems,
```

Comment:
I suggest changing "students" to "learners" for more inclusivity

Reply:
ooohhhh, nice thought, will do it right away. Thank You🤗